docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,69 +1,177 @@
+
+The following command starts a single-node {es} cluster for development or
+testing.
+
+. Start {es} in Docker. A password is generated for the `elastic` user and
+output to the terminal, plus an enrollment token for enrolling {kib}.
++
+--
 ifeval::["{release-state}"!="unreleased"]
-["source","yaml",subs="attributes"]
---------------------------------------------
-include::docker/docker-compose.yml[]
---------------------------------------------
+[source,sh,subs="attributes"]
+----
+docker run --name es-node01 -p 9200:9200 -p 9300:9300 -it docker.elastic.co/elasticsearch/elasticsearch:{version}
+----
+
 endif::[]
-// tag::docker-memory[]
-. Make sure Docker Engine is allotted at least 4GiB of memory.
-In Docker Desktop, you configure resource usage on the Advanced tab in Preference (macOS)
-or Settings (Windows).
-// end::docker-memory[]
+--
++
+TIP: You might need to scroll back a bit in the terminal to view the password
+and enrollment token.
+
+. Copy the generated password and enrollment token and save them in a secure
+location. These values are shown only when you start {es} for the first time.
++
+[NOTE]
+====
+If you need to reset the password for the `elastic` user or other
+built-in users, run the {ref}/reset-password.html[`elasticsearch-reset-password`]
+tool. To generate new enrollment tokens for {kib} or {es} nodes, run the
+{ref}/create-enrollment-token.html[`elasticsearch-create-enrollment-token`] tool.
+These tools are available in the {es} `bin` directory of the Docker container.
+For example:
+
+[source,sh]
+----
+docker exec -it es-node01 /usr/share/elasticsearch/bin/elasticsearch-reset-password
+----
+====
 
-. Run `docker-compose` to bring up the three-node {es} cluster and {kib}:
+. Copy the `http_ca.crt` security certificate from your Docker container to
+your local machine.
 +
-["source","sh",subs="attributes"]
---------------------------------------------
-docker-compose up
---------------------------------------------
+[source,sh]
+----
+docker cp es-node01:/usr/share/elasticsearch/config/tls_auto_config_*/http_ca.crt .
+----
 
-. Submit a `_cat/nodes` request to see that the nodes are up and running:
+. Open a new terminal and verify that you can connect to your {es}
+cluster by making an authenticated call, using the `http_ca.crt` file that you
+copied from your Docker container. Enter the password for the `elastic` user
+when prompted.
 +
 [source,sh]
---------------------------------------------------
-curl -X GET "localhost:9200/_cat/nodes?v&pretty"
---------------------------------------------------
+----
+curl --cacert http_ca.crt -u elastic https://localhost:9200
+----
+// NOTCONSOLE
 
-. Open {kib} to load sample data and interact with the cluster:
-http://localhost:5601.
+. In a new terminal session, start {kib} and connect it to your {es} container:
++
+[source,sh,subs="attributes"]
+----
+docker run --name kib-01 --net elastic -p 5601:5601 docker.elastic.co/kibana/kibana:{version}
```

Comment:
This command fails because the `elastic` network doesn't exist.

Reply:
I'll add `--net elastic` to the command for starting Elasticsearch on line 106.

Reply:
I'm not sure if this solves the problem, IIRC the network must be created explicitly.
Can we just use the default network instead?